File: Python/maxinlst.py
def max_val(lst):
    lst_max = lst[0]
    for i in range(len(lst)):
        if lst[i] > lst_max:
            lst_max = lst[i]
    return lst_max


if __name__ == "__main__":
    li = list(map(int, input("Please enter a list of integers: """).split()))
    m = max_val(li)
    print(f"Max value is {m}")

File: C++/maxinlst.cpp
#include <iostream>
using namespace std;

int maxinlst(int lst[], int size);


int main() {
    int test[] = {1,2,3,4,5,6};
    int max = maxinlst(test, 6);
    cout << max << endl;
    return 0;
}


int maxinlst(int lst[], int size) {
    int max = lst[0];
    for (int i = 0; i < size; i++) {
        if (max < lst[i]) {
            max = lst[i];
        }
    }
    return max;
}
